.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,119 @@
+name: " Bug Report"
```

Comment:
There are two bug report template files being added: `bug-report.yml` and `bug_report.yml`. These files have similar content but slight differences. This will cause confusion for users. Please consolidate these into a single bug report template.